Advanced-Audio-Driver-Java/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,8 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
@jintgeorge without this, permissions could not be validated in AdvancedAudioDevice.java file